### ISSUE TITLE
connecting multiple loras to same KSampler without merging their effects

### ISSUE THREAD

**Issue by loboere** [comfyanonymous/ComfyUI#1247]
I'm struggling to prevent blending the effects of multiple loras in Comfy, without success so far... is there a better way to connect the loras to the KSampler to prevent them from cancelling each other?

I want subjects to keep 100% their original lora properties.

**Comment by asagi4** [comment]
Can you explain what you mean by this? LoRAs are applied to the model, not in the sampler, so your question doesn't quite make sense. Do you mean you want the sampler to be able to use different LoRAs for different regions of the image? You can do that at the latent level by just combining latents from multiple samplers.

**Comment by loboere** [comment]
@asagi4 I have two loras for two different people, if I use the loras individually the people are recognizable, but if I use it together, the people are not recognizable, rather they look like a mixture of the two people as if they were their son

**Comment by asagi4** [comment]
Yeah, you'll have to generate two latents and combine them. There's no way at least in core ComfyUI to separate the effects of two LoRAs in a single sampler.

**Comment by loboere** [comment]
can you show me an example workflow
